### ISSUE TITLE
Crate builder 0.25.0 released

### ISSUE THREAD

**Issue by marcolarosa** [describo/crate-builder-component-react#2]
https://github.com/describo/crate-builder-component/releases/tag/v0.25.0

**Comment by marcolarosa** [comment]
Thanks for the pull request and i've moved the navigation emitter to the start of the method. That was a bug. Navigation is always emitted. It's up to the outer app to listen to it if it wants to. But if it does, it should probably set `enableInternalRouting` to false.

New version 0.25.1 published to npm.

**Comment by beepsoft** [comment]
Thanks, it works now!

I thought you pass the `@id` of the entity along with the navigation event, but it seems to be some kinfo internal id. 

For example, when I navigate to this entity:

```json
        {
            "@id": "http://nla.gov.au/nla.party-479603",
            "@type": "Person",
            "email": "thien@unimelb.edu.au",
            "familyName": "Thieberger",
            "givenName": "Nick",
            "homeLocation": { "@id": "#country_Australia" },
            "name": "Nick Thieberger"
        }
```

I expected to get `http://nla.gov.au/nla.party-479603` as the id argument, but I got `od3upvwyduhee8yy5z34538e7z5m25ol`

What is this id usable for? Could you maybe also pass the `@id` of the entity where we navigate to?

**Comment by marcolarosa** [comment]
Yeah - that is an internal id and that's the one I use in the location because it saves me having to url encode the @id property and dealing with any of that complexity. Plus, it maps directly to internal entity representation. Anyway, I've just published 0.25.2 which also emits `@id` in the navigation event so you can use that if you prefer.

**Comment by beepsoft** [comment]
Thanks, it works great, published the new release: https://www.npmjs.com/package/@describo/crate-builder-component-react
